.github/workflows/ci.yml

```diff
@@ -24,4 +24,10 @@ jobs:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
     - run: bin/setup
+      # Skip to test with Ruby HEAD because rbs-3.9.4 crashes with Ruby 3.5.0
+      # ref: https://github.com/ruby/rbs/pull/2537/commits/c9389681acf10c9fdd34601e74153011919b8544
+      if: matrix.ruby != 'head'
```

Comment:
I also met testing with Ruby HEAD is also broken. So I determined to skip it for a while.